13-template.tex

```diff
@@ -31,8 +31,8 @@ \chapter{Analysis}
 
 \section{Requirements}
 
-The goal of the software HOMER (HOMe EmulatoR) is to emulate a domotic controller connected to lights, windows, and other domestic devices.
-The software will provide a dashboard to the user to allow control of the devices and monitoring of sensors and consumptions.
+HOMER (HOMe EmulatoR) is an emulated domotic controller connected to lights, windows, and other domestic facilities.
+The smart environment is mananged through a dashboard which allows the user to control devices, monitor sensors and consumptions.
```

Comment:
Typo "mananged" should be corrected to "managed"

Reply:
will fix ASAP